src/main/resources/google_checks.xml

```diff
@@ -105,6 +105,12 @@
       <property name="query" value="//RCURLY[parent::SLIST[count(./*)=1]
                                      or preceding-sibling::*[last()][self::LCURLY]]"/>
     </module>
+    <module name="SuppressionXpathSingleFilter">
+      <property name="checks" value="MethodName"/>
+      <property name="query" value="//METHOD_DEF[
+                                     .//ANNOTATION/IDENT[@text='Test']]//*"/>
+      <property name="message" value=".*_.*"/>
```

Comment:
Let's make message match to be
`.*'[a-z][a-z0-9][_a-zA-Z0-9]*'.*`
It should not suppress your case `@Test  void Testing_Foo_False_Negative()`

Reply:
done, changed regex a little bit, explained here: https://github.com/checkstyle/checkstyle/pull/15515#discussion_r1720790213